packages/viz/src/LineChart/LineChart.stories.tsx

```diff
@@ -798,9 +798,28 @@ export const CustomEchartsOptions: StoryObj<HvLineChartProps> = {
         groupBy="Month"
         measures="Sales Target"
         onOptionChange={(option) => {
+          console.log(option);
           if (Array.isArray(option.yAxis) && option.yAxis.length === 1) {
             option.yAxis = [{ ...option.yAxis[0], splitNumber: 8 }];
           }
+          option.series[0].markLine = {
+            data: [{ yAxis: 3000, name: "Average" }],
+            symbol: "none",
+            itemStyle: {
+              color: "#333333",
```

Comment:
This isn't visible in wicked 😬
![image](https://github.com/lumada-design/hv-uikit-react/assets/638946/0f1cb3a9-cbee-45eb-a84b-e80e75aadbe7)


Can't we use colors from the palette? If not, we could use a non-gray color (eg. orange?)